scripts/debug_tools/prepare_all_cmd_for_ctu.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+import argparse
+import json
+import os
+import subprocess
+import sys
+
+import prepare_compile_cmd
```

Comment:
What are these imports, where do they come from?
If they come from the current package, I would rather see `from . import xyz` so it is not mixed up with a "system" (or virtualenv) import on first glance.

Reply:
As we discussed it will not work.